spec/lib/video_info_spec.rb

```diff
@@ -25,6 +25,29 @@
     end
   end
 
+  describe ".disable_providers" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.